earthaccess/auth.py

```diff
@@ -293,33 +316,41 @@ def _find_or_create_token(self, username: str, password: str) -> Any:
 
     def _persist_user_credentials(self, username: str, password: str) -> bool:
         # See: https://github.com/sloria/tinynetrc/issues/34
+
+        netrc_loc = netrc_path()
+        logger.info(f"Persisting credentials to {netrc_loc}")
+
         try:
-            netrc_path = Path().home().joinpath(".netrc")
-            netrc_path.touch(exist_ok=True)
-            netrc_path.chmod(0o600)
+            netrc_loc.touch(exist_ok=True)
+            netrc_loc.chmod(0o600)
         except Exception as e:
             logger.error(e)
             return False
-        my_netrc = Netrc(str(netrc_path))
+
+        my_netrc = Netrc(str(netrc_loc))
         my_netrc[self.system.edl_hostname] = {
             "login": username,
             "password": password,
         }
         my_netrc.save()
+
         urs_cookies_path = Path.home() / ".urs_cookies"
+
         if not urs_cookies_path.exists():
             urs_cookies_path.write_text("")
 
         # Create and write to .dodsrc file
         dodsrc_path = Path.home() / ".dodsrc"
+
         if not dodsrc_path.exists():
```

Comment:
Is this for OPeNDAP? amazing!

Reply:
I suppose so, but this was _existing_ code. I just added a bit of whitespace for readability.